app/scripts/templates/visits/item.html

```diff
@@ -1,4 +1,12 @@
-<div class="visit">
+<div class="image" style="background-image: url({{extractedImageUrl}});"></div>
+<div class="text">
+  <div class="provider">
+    <img class="favicon" src="{{faviconUrl}}" />
```

Comment:
Not sure if you want to add `alt="{{extractedProviderName}}"` for screen readers or whatevers.